automation/UNIX/test-lnt.sh

```diff
@@ -0,0 +1,36 @@
+# Test clang on UNIX using Visual Studio Team Services
+
+set -ue
+set -o pipefail
+set -x
```

Comment:
It's helpful to have the detailed trace of commands when there are automation failures.   It does make the logs harder to read because they are more verbose, but I don't think it is excessive in this case (the results are at the end of the log file).